pkg/service/query.go

```diff
@@ -90,3 +95,8 @@ func getTypeForQueryService(jaeger *v1.Jaeger) corev1.ServiceType {
 	}
 	return corev1.ServiceTypeClusterIP
 }
+
+// GetNodePortForQueryService returns the query service NodePort for this Jaeger instance
+func GetNodePortForQueryService(jaeger *v1.Jaeger) int {
```

Comment:
Why this function is returning int instead of int32. When you use it you cast it again to int32
I would suggest to change this to:
```
func GetNodePortForQueryService(jaeger *v1.Jaeger) int32 {
	return jaeger.Spec.Query.NodePort
}
```
And ged rid of all `int32(GetNodePortForQueryService(jaeger))` casts.

Reply:
Hi -
I set the function to return an `int` in lieu of an `int32`, for consistency with the `getTargetPortForQueryService` function. Let me make the change and update this MR 👍